source/MRMesh/MRSymbolMesh.cpp

```diff
@@ -214,7 +214,7 @@ Contours2d createSymbolContours( const SymbolMeshParams& params )
 Mesh triangulateSymbolContours( const SymbolMeshParams& params )
 {
     MR_TIMER
-    return PlanarTriangulation::triangulateContours( createSymbolContours( params ) );
+    return *PlanarTriangulation::triangulateContours( createSymbolContours( params ) );
```

Comment:
Please always check that option is not nullopt

Reply:
reworked